### PR TITLE
Editing pass for repeat.adoc

### PR DESCRIPTION
**Pull request by Buzzardo**
I edited for readability and consistency and fixed sentence errors. No questions this time.

### CODE REVIEW

File: spring-batch-docs/asciidoc/repeat.adoc [hunk 1]
```diff
@@ -75,31 +74,31 @@ template.iterate(new RepeatCallback() {
...
+      ignore the context. However, if necessary, it can be used as an attribute bag
       to store transient data for the duration of the iteration. After the
-      `iterate` method returns, the context will no
-      longer exist.
+      `iterate` method returns, the context no
+      longer exists.
 
-A `RepeatContext` will have a parent context
-      if there is a nested iteration in progress. The parent context is
+if there is a nested iteration in progress, a `RepeatContext` has a parent context.
```
Comment (cppwfs) [inline spring-batch-docs/asciidoc/repeat.adoc]: If needs to be capitalized.  If you agree I can fix on merge.

File: spring-batch-docs/asciidoc/repeat.adoc [hunk 2]
```diff
@@ -167,9 +166,10 @@ Users might need to implement their own completion policies for more
 If there is an exception thrown inside a
     `RepeatCallback`, the
     `RepeatTemplate` consults an
-    `ExceptionHandler` which can decide whether or not to
+    `ExceptionHandler`, which can decide whether or not to
     re-throw the exception.
 
+The following listing shows the `ExceptionHandler` interface definition:
 
 [source, java]
```
Comment (cppwfs) [inline spring-batch-docs/asciidoc/repeat.adoc]: Noticed that handleException throws a Throwable not a RuntimeException.   
I can change this on Merge

File: spring-batch-docs/asciidoc/repeat.adoc (hunk 1 above)
Comment (Buzzardo) [reply spring-batch-docs/asciidoc/repeat.adoc]: Yes, please. Thanks, Glenn.

File: spring-batch-docs/asciidoc/repeat.adoc (hunk 2 above)
Comment (Buzzardo) [reply spring-batch-docs/asciidoc/repeat.adoc]: Yes, please. Thanks again.

**Comment by cppwfs** [comment]
LGTM!

**Comment by cppwfs** [comment]
Updated doc with fixes mentioned above.   Rebased, Squashed, Merged

**Pull request closed without merging**